README.md

```diff
@@ -1,32 +1,249 @@
-# Introduction to Threads
+# What is a Thread?
 
 ## Learning Goals
 
-- Learning Goal 1
-- Learning Goal 2
+- Explain the differences between a program, process, and thread.
+- Define the relationship between a thread and a stack.
+- Demonstrate single threaded versus multithreaded program execution.
 
 ## Introduction
 
-A 1-2 sentence summary of what will be covered.
+In order to understand concurrency and multithreading we have to first
+understand some basic terms. We’ll look at the following terms:
 
-## Topic 1
+- Program
+- Process
+- Thread
+- Stack
 
-Discuss the topic of the reading. Break into sections and sub-sections as
-appropriate to help with the reader's orientation with the material and future
-referencing.
+A program is a set of instructions and data to perform a specialized task.
```

Comment:
Suggestion: Possibly bold the terms we are defining to make them stand out a little more?

Reply:
good idea, thanks